test/WebJobs.Script.Tests/Workers/Rpc/RpcWorkerProcessTests.cs

```diff
@@ -18,39 +20,34 @@ namespace Microsoft.Azure.WebJobs.Script.Tests.Workers.Rpc
 {
     public class RpcWorkerProcessTests
     {
-        private readonly Mock<IWorkerProcessFactory> _workerProcessFactory;
-        private RpcWorkerProcess _rpcWorkerProcess;
-        private Mock<IScriptEventManager> _eventManager;
-        private Mock<IHostProcessMonitor> _hostProcessMonitorMock;
-        private TestLogger _logger = new TestLogger("test");
-        private IOptions<FunctionsHostingConfigOptions> _functionsHostingConfigOptions;
+        private readonly Mock<IWorkerProcessFactory> _workerProcessFactory = new Mock<IWorkerProcessFactory>();
```

Comment:
Are these changes unrelated / needed? Can we fix the inconsistencies in private variable names (some has _ prefix and some does not)

Reply:
These came from another comment in PR to refactor the code so that we don't create 2 instances of `RpcWorkerProcess`.
Updated the private naming in next iteration